jekyll/_cci2/sample-config.md

```diff
@@ -327,746 +155,103 @@ workflows:
-            - image: node:current-alpine
-              auth:
-                username: mydockerhub-user
-                password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-        steps:
-            - attach_workspace:
-                  at: .
-            - run:
-                  name: Production build
-                  command: |
-                      export __BUILD_VERSION="$(cat version.txt)"
-                      yarn build
-            - store_artifacts:
-                  path: dist/server.js
-            - persist_to_workspace:
-                  root: .
-                  paths:
-                      - .
+This example shows a sequential workflow with the `test` job configured to run only on the main branch. Refer to the [Workflows]({{ site.baseurl }}/workflows) document for complete details about orchestrating job runs with concurrent, sequential, and manual approval workflows.
 
-    build-docker-image:
-        machine:
-            # The image uses the current tag, which always points to the most recent
-            # supported release. If stability and determinism are crucial for your CI
-            # pipeline, use a release date tag with your image, e.g. ubuntu-2004:202201-02
-            image: ubuntu-2004:current
-        steps:
-            - attach_workspace:
-                  at: .
-            - run:
-                  name: Setup __BUILD_VERSION envvar
-                  command: |
-                      echo 'export __BUILD_VERSION="$(cat version.txt)"' >> "$BASH_ENV"
-            - docker/check:
-                  registry: $DOCKER_REGISTRY
-            - docker/build:
-                  image: $DOCKER_IMAGE_NAME
-                  tag: $__BUILD_VERSION
-                  registry: $DOCKER_REGISTRY
-            - docker/push:
-                  image: $DOCKER_IMAGE_NAME
-                  tag: $__BUILD_VERSION
-                  registry: $DOCKER_REGISTRY
+## Sample configuration with fan-in/fan-out workflow
+{: #sample-configuration-with-fan-infan-out-workflow }
+Below are two sample configurations for a Fan-in/Fan-out workflow.
 
-    test:
-        docker:
-            - image: node:current-alpine
-              auth:
-                username: mydockerhub-user
-                password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-        parallelism: 2
-        steps:
-            - attach_workspace:
-                  at: .
-            - run:
-                  name: Run tests
-                  command: |
-                      circleci tests glob '**/*.test.ts' | circleci tests split --split-by timings | xargs yarn test:ci
-            - store_artifacts:
-                  path: test-results
-            - store_test_results:
-                  path: test-results
+For the Server/`2.0` config example, refer to [the complete demo repo on GitHub](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/fan-in-fan-out/.circleci/config.yml) for details.
```

Comment:
Do we mean to keep a reference to a 2.0 config example?

Reply:
Oh good catch!